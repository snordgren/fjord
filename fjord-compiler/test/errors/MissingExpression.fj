module test.errors.MissingExpression

empty : Int
empty = 
