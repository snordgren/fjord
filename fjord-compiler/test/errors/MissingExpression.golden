test/errors/MissingExpression.fj:5:2:
  |
5 |  
  |  ^
unexpected end of input
expecting expression
